packages/web-pkg/src/composables/router/types.ts

```diff
@@ -0,0 +1,8 @@
+
+type Dictionary<T> = { [key: string]: T }
```

Comment:
This is a not exported type from `vue-router`, the other types here basically match the (unnamed) types in the `Location` interface of `vue-router`.

Reply:
well, they are unnamed and hence even less exported :-) 
https://github.com/vuejs/vue-router/blob/d76980c5dd26e1d4df68e7154929af7264614fb2/types/router.d.ts#L189

Sorry, the comment was probably a bit misleading... I just meant I gave reusable typenames to something like `params?: Dictionary<string>`